### PR TITLE
get_missing_values now computes number of instances with missing inst…

### LINKED ISSUE #46: fix get_missing_values function
as it is right now, get_missing_values returns ratio/number of features instead of the ratio/number of instances

### PR DESCRIPTION
**Pull request by macetheace96**
…ead of number of features. Fixes #46

### CODE REVIEW

File: metalearn/metafeatures/simple_metafeatures.py [hunk 1]
```diff
@@ -18,11 +18,11 @@ def get_dimensionality(number_of_features, number_of_instances):
...
 
 def get_missing_values(X):
-    missing_values = X.shape[0] - X.count()
-    number_missing = np.sum(missing_values)
+    missing_values_by_instance = X.shape[1] - X.count(axis=1)
+    number_missing = np.sum(missing_values_by_instance)
     ratio_missing = float(number_missing) / float(X.shape[0] * X.shape[1])
-    number_instances_with_missing = X.shape[1] - np.sum(missing_values == 0)
-    ratio_instances_with_missing = float(number_instances_with_missing) / float(X.shape[1])
+    number_instances_with_missing = X.shape[0] - np.sum(missing_values_by_instance == 0)
```
Comment (bjschoenfeld) [inline metalearn/metafeatures/simple_metafeatures.py]: Could `np.sum(missing_values_by_instance != 0)` work?

File: metalearn/metafeatures/simple_metafeatures.py [hunk 2]
```diff
@@ -18,11 +18,11 @@ def get_dimensionality(number_of_features, number_of_instances):
...
+    ratio_instances_with_missing = float(number_instances_with_missing) / float(X.shape[0])
```
Comment (bjschoenfeld) [inline metalearn/metafeatures/simple_metafeatures.py]: The `float()`'s could be removed. In python 3, division is already float division.

Comment (macetheace96) [reply metalearn/metafeatures/simple_metafeatures.py]: should we take them off of all the functions in simple_metafeatures? I was just copying the syntax

Comment (bjschoenfeld) [reply metalearn/metafeatures/simple_metafeatures.py]: I think so.

**Comment by macetheace96** [comment]
it should be good now

**Pull request merged**